README.md

```
@@ -102,6 +102,34 @@ pnpm authhero      # Work with main package
 pnpm vitepress     # Work with docs
 ```
 
+### Running OIDC Conformance Tests
+
+AuthHero is tested against the [OpenID Foundation conformance suite](https://gitlab.com/openid/conformance-suite) via [`apps/conformance-runner`](apps/conformance-runner/), a Playwright-driven runner that boots the suite, seeds a local auth-server, and runs the `oidcc-basic-certification-test-plan`.
+
+**One-time setup:**
+
+1. Clone the conformance suite into `~/conformance-suite` (the scripts expect this path).
+2. Add the suite's hostname to `/etc/hosts`:
+   ```
```

markdownlint-cli2 (0.22.1)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)